musabase/cgiar_banana_ontology.obo

```diff
@@ -5706,6 +5706,229 @@ namespace: BananaTrait
+relationship: variable_of CO_325:0010263 ! outer damage at 10 cm - measurement
+relationship: variable_of CO_325:0100046 ! %
+
+[Term]
+id: CO_325:0003011
+name: Inner damage percentage at 5 cm from the stellar region after weevil introduction
+namespace: BananaTrait
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010266 ! inner damage at 5 cm - measurement
+relationship: variable_of CO_325:0100046 ! %
+
+[Term]
+id: CO_325:0003012
+name: Inner damage percentage at 10 cm from the stellar region after weevil introduction
+namespace: BananaTrait
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010265 ! inner damage at 10 cm - measurement
+relationship: variable_of CO_325:0100046 ! %
+
+[Term]
+id: CO_325:0003014
+name: nematode damage
+namespace: BananaTrait
+is_a: CO_325:1000002 ! Biotic_stress
+
+[Term]
+id: CO_325:0003015
+name: Helicotylincus mulicinctus damage
+namespace: BananaTrait
+def: "Helicotylincus mulicinctus damage assesssment" []
+is_a: CO_325:0003014 ! nematode damage
+
+[Term]
+id: CO_325:0003016
+name: Number of female Helicotylincus mulicinctus - counting
+namespace: BananaTrait
+def: "Measurement per 100 grams of soil sample" []
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003015 ! Helicotylincus mulicinctus damage
+relationship: variable_of CO_325:0010223 ! number of nematodes (helicotylenchus multicinctus) per unit of soil sample - method
+relationship: variable_of CO_325:0100102 ! unitless
+
+[Term]
+id: CO_325:0003017
+name: Number of male Helicotylincus mulicinctus - counting
+namespace: BananaTrait
+def: "Measurement per 100 grams of soil sample" []
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003015 ! Helicotylincus mulicinctus damage
+relationship: variable_of CO_325:0010223 ! number of nematodes (helicotylenchus multicinctus) per unit of soil sample - method
+relationship: variable_of CO_325:0100102 ! unitless
+
+[Term]
+id: CO_325:0003018
+name: Number of juvenile Helicotylincus mulicinctus - counting
+namespace: BananaTrait
+def: "Measurement per 100 grams of soil sample" []
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003015 ! Helicotylincus mulicinctus damage
+relationship: variable_of CO_325:0010223 ! number of nematodes (helicotylenchus multicinctus) per unit of soil sample - method
+relationship: variable_of CO_325:0100102 ! unitless
+
+[Term]
+id: CO_325:0003019
+name: Radopholus similis damage
+namespace: BananaTrait
+def: "Radopholus similis damage assesssment" []
+is_a: CO_325:0003014 ! nematode damage
+
+[Term]
+id: CO_325:0003020
+name: Number of female Radopholus similis - counting
+namespace: BananaTrait
+def: "Measurement per 100 grams of soil sample" []
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003019 ! Radopholus similis damage
```

Comment:
shouldn't it be linked to id: CO_325:0000524
name: number of rotylenchulus spp.per soil sample instead

Reply:
CO_325:0000522
number of radopholus similis per soil sample